scripts/data_utils.py

```diff
@@ -489,10 +668,15 @@ def chunk_content_helper(
                 splitter = PythonCodeTextSplitter.from_tiktoken_encoder(
                     chunk_size=num_tokens, chunk_overlap=token_overlap)
             else:
-                splitter = RecursiveCharacterTextSplitter.from_tiktoken_encoder(
-                    separators=SENTENCE_ENDINGS + WORDS_BREAKS,
-                    chunk_size=num_tokens, chunk_overlap=token_overlap)
+                if file_format == "html_pdf": # cracked pdf converted to html

```

Comment:
do we need to convert it back to pdf after chunking. chunk.filename = chunk.filename.replace(".html_pdf", ".pdf")?

Reply:
From my understanding, the original file_path remains unchanged and is used later rather than the file_format which is only used during chunking.